stakewise_cli/commands/sync_validator_keys.py

```diff
@@ -81,6 +82,11 @@ def sync_validator_keys(
     with open(join(output_dir, LIGHTHOUSE_CONFIG_FILENAME), "w") as f:
         f.write(lighthouse_config)
 
+    # save external signer public keys
+    signer_keys_config = _generate_signer_keys_config(public_keys=keys)
```

Comment:
I suggest adding a comment or renaming the function so that it would be more clear what clients it is used for